src/main/java/com/alipay/oceanbase/rpc/ObTableClient.java

```diff
@@ -2570,13 +2690,25 @@ public ObPayload execute(final ObTableAbstractOperationRequest request) throws E
                 .executeInternal();
             return batchOpsResult.getResults().get(0);
         } else if (request instanceof ObTableQueryRequest) {
-            ObTableClientQueryImpl tableQuery = new ObTableClientQueryImpl(request.getTableName(),
+            // TableGroup -> TableName
+            String tableName = request.getTableName();
+            if (((ObTableQueryRequest) request).getTableQuery().isHbaseQuery() 
+                    && isTableGroupName(tableName)) {
+                tableName = tryGetTableNameFromTableGroupCache(tableName, false);
```

Comment:
When will you replace the realTableName with tableGroupName

Reply:
![image](https://github.com/oceanbase/obkv-table-client-java/assets/23191541/7d844cb5-bd0f-43b8-b60d-18c79d5d50f0)